installer/conf/sudoers

```diff
@@ -34,6 +34,9 @@ omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsagent/ruby/bin/ruby /opt/microsof
 # Enable nxOMSAuditdResource
 omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsconfig/Scripts/OMSAuditdPlugin.sh
 
+#Enable shutdown for update management
+omsagent ALL=(ALL) NOPASSWD: /sbin/shutdown -r *
```

Comment:
Avoid asterisks in the sudoers file if it is not needed. `shutdown` is usually called with `now` as a parameter.